modules/common/adapters/adapter_gflags.cc

```diff
@@ -40,6 +40,8 @@ DEFINE_string(control_preprocessor_topic, "/apollo/control/preprocessor",
               "control preprocessor topic name");
 DEFINE_string(control_local_view_topic, "/apollo/control/localview",
               "control local view topic name");
+DEFINE_string(control_pre_command_topic, "/apollo/control/precommand",
```

Comment:
How it is defined? Try match the proto definition with topic definition to avoid confusion.